tests/wsl/firstrun.pm

```diff
@@ -8,17 +8,23 @@
 # without any warranty.
 
 # Summary: Configure WSL users
-# Maintainer: Martin Loviska <mloviska@suse.com>
+# Maintainer: qa-c  <qa-c@suse.de>
 
-use base "windowsbasetest";
-use strict;
-use warnings;
+use Mojo::Base qw(windowsbasetest);
 use testapi;
-use version_utils 'is_sle';
+use version_utils qw(is_sle);
+
+sub is_sut_dev {
```

Comment:
I understand `is_sut_dev` means is "system under test under development" but it's not self explained... shall we say `is_beta` ?

Reply:
So I have failed as it is not self-explanatory. But it is exactly that case you have described. `is_beta` has crossed my mind but is not really indicating that the system is in dev phase.

Reply:
If you do not wanna add more conditions here. I also see it more obscured then clarified by this function.